src/mechanics/elements/Element.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include "mechanics/elements/ElementInterface.h"
+#include "mechanics/nodes/DofContainer.h"
+
+namespace NuTo
+{
+class Element
```

Comment:
Naming again: an `Element` does not derive from `ElementInterface`, and contains several "elements"? Could be confusing. Maybe `ElementStack`, `ElementDeck`, `ElementPack`, `ElementCollection` or `ElementAggregate`?

Reply:
I agree. How about renaming `ElementInterface` instead? `SingleDofElementInterface` or something. I think its nice to have one `Element` at the `Cell`.

Reply:
But you have several. The `ElementInterface` elements are named correctly imho.